frontend/ui/widget/inputdialog.lua

```diff
@@ -723,6 +724,45 @@ function InputDialog:_addScrollButtons(nav_bar)
                 end,
             })
         end
+        -- Add a button to go to the line by its number in the file
+        if self.fullscreen then
+            table.insert(row, {
+                text = _("Go"),
+                callback = function()
+                    local cur_line_num, last_line_num = self._input_widget:getLineNums()
+                    local input_dialog
+                    input_dialog = InputDialog:new{
+                        title = _("Enter line number"),
+                        input_hint = T("%1 (1 - %2)", cur_line_num, last_line_num),
```

Comment:
Should probably be this?
```suggestion
                        input_hint = T(_("%1 (1 - %2)"), cur_line_num, last_line_num),
```

Reply:
Will require above it:
`-- @translators %1 is the current line number, %2 is the last line number`

Reply:
How does his string vary in other language?

Reply:
Chances are it won't, but that's not the point.

Reply:
Fixed